homeassistant/components/laundrify/model.py

```diff
@@ -0,0 +1,13 @@
+"""Models for laundrify platform."""
+from __future__ import annotations
+
+from typing import TypedDict
+
+
+class LaundrifyDevice(TypedDict):
```

Comment:
The library should preferably define its model, eg with a dataclass, so we don't need to define the type here but can use the type from the library.